client/lib/menu-data/menu-data.js

```diff
@@ -870,7 +876,9 @@ MenuData.prototype.fetchDefaultMenu = function() {
 	this.data.defaultMenu = false;
 	this.fetchingDefaultMenu = true;
 
-	wpcom.site( requestedSiteID ).postsList( params, function( error, data ) {
+	wpcom
+	.site( requestedSiteID )
```

Comment:
I think it is.  Also, long lines should be avoided.

I'd prefer to indent the subsequent lines:

```
wpcom
    .site( .. )
    .postsList(
        ..
    );
```

but we use the un-indented style in lots of places.